src/main/java/winstone/HostConfiguration.java

```diff
@@ -117,7 +113,9 @@ private void loadBuiltinMimeTypes() {
             Properties props = new Properties();
             props.load(in);
             for (Entry<Object, Object> e : props.entrySet()) {
-                mimeTypes.addMimeMapping(e.getKey().toString(), e.getValue().toString());
+                this.server
+                        .getMimeTypes()
```

Comment:
A few lines above on line 96, you don't qualify the access to the `server` field (`server.getMimeTypes().addMimeMapping(extension.toLowerCase(), mimeType);`), so for consistency I don't think we should qualify the access here either:

```suggestion
                server.getMimeTypes()
```